R/tribble_paste.R

```diff
@@ -151,6 +151,8 @@ pad_to <-function(char_vec, char_length){
   paste0(strrep(" ",char_length - nchar(char_vec)),char_vec)
 }
 
+trim <- function (x) gsub("^\\s+|\\s+$", "", x)
```

Comment:
That would be perfectly acceptable, and I am somewhat confused over why I didn't know about it.